pandas/core/series.py

```diff
@@ -1374,15 +1374,20 @@ def corr(self, other, method='pearson', min_periods=None):
         return nanops.nancorr(this.values, other.values, method=method,
                               min_periods=min_periods)
 
-    def cov(self, other, min_periods=None):
+    def cov(self, other, min_periods=None, ddof=1):
         """
         Compute covariance with Series, excluding missing values
+        
+        Normalized by N-1 by default. This can be changed using the ddof argument"
```

Comment:
Can probably make this a new Notes section, and combine with the note that's currently in the Returns section.